packages/components/src/border-control/types.ts

```diff
@@ -78,9 +85,7 @@ export type BorderControlProps = ColorProps &
 		 */
 		shouldSanitizeBorder?: boolean;
 		/**
-		 * Whether or not to show the header for the border color and style
-		 * picker dropdown. The header includes a label for the color picker
-		 * and a close button.
+		 * @deprecated This prop no longer has any effect.
```

Comment:
Like @ciampo [suggested](https://github.com/WordPress/gutenberg/pull/65475#discussion_r1778414108), we should add an `@ignore` to suppress this prop from the Storybook props table.

Reply:
Ah right, forgot about that, will do thanks!